storage/disabled/storer.go

```diff
@@ -0,0 +1,85 @@
+package disabled
+
+import (
+	storageCore "github.com/ElrondNetwork/elrond-go-core/storage"
+)
+
+type storer struct{}
+
+// NewStorer returns a new instance of this disabled storer
```

Comment:
Oh, we didn't have an existing "generic" disabled storer, all the other ones were defined in their client packages. :+1: for adding it here.